packages/key-management/src/util/KeyAgentTransactionSigner.ts

```diff
@@ -32,6 +32,7 @@ export class KeyAgentTransactionSigner implements TransactionSigner {
       tx,
       {
         knownAddresses: [],
+        purpose: this.#account.purpose,
```

Comment:
It doesn't seem like `Bip32Account.purpose` is used anywhere else, so we can probably undo changes in `Bip32Account.ts`
```suggestion
        purpose: this.#keyAgent.purpose,
```

Reply:
Fixed on 7c3a32f